client/dist/style.css

```diff
@@ -86,23 +74,131 @@
   padding-top: 25px;
 }
 
-.colored {
-  color: #da3743;
-}
+/*CSS for stars*/
 
 .stars-component {
   display: flex;
   flex-direction: row;
-  padding-bottom: 10px;
+  padding-bottom: 4%;
+}
+
+.colored {
+  color: #da3743;
 }
 
 .uncolored {
   color: #6f737b;
 }
 
-.icon-container {
+/*CSS for header*/
+
+.header-container {
+  font-family: 'Josefin Sans', sans-serif;
+  font-weight: bold;
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-top: 15px;
+  border-bottom: 1px solid #2d333f;
+  border-top: 1px solid #2d333f;
+  padding-left: 5%;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.average-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.avg-rating-block {
+  margin-right: 10px;
+}
+
+.header-icon-container {
+  margin-bottom: 5px;
+}
+
+.award {
+  margin-top: 10px;
+  border: 1px solid #2d333f;
+  height: 50px;
+  width: 300px;
+  border-radius: 8px;
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.award:hover {
+  border: 5px solid #da3743;
+  outline: none;
+}
+
+.award-text {
+  margin-top: 10px;
+}
+
+/* CSS for bargraph */
+
+.bargraph {
   display: block;
   position: absolute;
-  bottom: 11px;
-  right: 16px;
-}
\ No newline at end of file
+  top: 94px;
+  left: 60%;
+}
+
+.header-info {
+  max-width: 50%;
+}
+
+@media all and (max-width: 975px) and (min-width: 300px) {
+  .header-info {
+    max-width: 20%;
+  }
+}
+
+.bar-number {
+  font-family: 'Josefin Sans', sans-serif;
+  font-weight: bold;
+}
+
+
+.bar-line {
+  height: 20px;
+  width: 300px;
+  margin-bottom: 4px;
+  border: 1px solid #2d333f;
+}
+
+.bar-line-red {
+  background: #da3743;
+  height: 20px;
+}
+
+.emphasize {
+  font-weight: bold;
+}
+
+/*Sorting CSS*/
+.sorting-text{
+  font-family: 'Josefin Sans', sans-serif;
+  margin-left: 10%;
+  margin-top: 20px;
+  font-size: 20px;
+}
+
+.dropdown {
+  width: 200px;
+  height: 30px;
+  margin-left: 12px;
+  font-family: 'Josefin Sans', sans-serif;
```

Comment:
Do you have multiple different fonts in your component? If you're only using 'Josefin Sans', you can just set font-family in body { } and the rest of the page will inherit it